utilities/transactions/write_unprepared_transaction_test.cc

```diff
@@ -188,6 +188,54 @@ TEST_P(WriteUnpreparedTransactionTest, ReadYourOwnWrite) {
   delete txn;
 }
 
+TEST_P(WriteUnpreparedTransactionTest, RecoveryRollbackUnprepared) {
```

Comment:
To the test rollback logic further, we can have another test that starts with a non-empty db, and checks that the DB state is the same after recovery.